src/pages/statement-of-support.md

```diff
@@ -68,7 +72,7 @@ Customers on our *Summit* plan, for major updates, can request a deferred upgrad
 
 ### Infrastructure Reviews
 
-As part of our Infrastructure Review service, our team review your current infrastructure and make recommendations in the following areas:
+As part of our [Infrastructure Review service](https://snowplow.io/snowplow-bdp-product-description/#:~:text=Infrastructure%20Management-,Regular%20Infrastructure%20Reviews,-Deferred%20Upgrades), our team review your current infrastructure and make recommendations in the following areas:
```

Comment:
I don’t think the link works, or at least it does not seem to lead to a specific header

Reply:
Fixed this. Thanks for flag @stanch. Also rebased so should be good to go. 🙏🏻